main/acle.md

```diff
@@ -9546,25 +9680,2141 @@ possible to write these operations using normal C arithmetic. For example:
+                                       svfloat32_t zm);
+  
+  ```
+
+#### FMLA, FMLS (multi)
+
+Multi-vector floating-point fused multiply-add/subtract
+
+``` c
+  // Variants are available for:
+  //   _za32[_f32]
+  //   _za64[_f64] (only if __ARM_FEATURE_SME_F64F64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmla_za32[_f32]_vg1x2(uint32_t slice, svfloat32x2_t zn,
+                              svfloat32x2_t zm);
+  
+  
+  // Variants are available for:
+  //   _za32[_f32]
+  //   _za64[_f64] (only if __ARM_FEATURE_SME_F64F64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmla_za32[_f32]_vg1x4(uint32_t slice, svfloat32x4_t zn,
+                              svfloat32x4_t zm);
+  
+  
+  // Variants are available for:
+  //   _za32[_f32]
+  //   _za64[_f64] (only if __ARM_FEATURE_SME_F64F64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmls_za32[_f32]_vg1x2(uint32_t slice, svfloat32x2_t zn,
+                              svfloat32x2_t zm);
+  
+  
+  // Variants are available for:
+  //   _za32[_f32]
+  //   _za64[_f64] (only if __ARM_FEATURE_SME_F64F64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmls_za32[_f32]_vg1x4(uint32_t slice, svfloat32x4_t zn,
+                              svfloat32x4_t zm);
+  
+  ```
+
+#### FMLA, FMLS (indexed)
+
+Multi-vector floating-point fused multiply-add/subtract
+
+``` c
+  // Variants are available for:
+  //   _za32[_f32]
+  //   _za64[_f64] (only if __ARM_FEATURE_SME_F64F64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmla_lane_za32[_f32]_vg1x2(uint32_t slice, svfloat32x2_t zn,
+                                   svfloat32_t zm, uint64_t imm_idx);
+  
+  
+  // Variants are available for:
+  //   _za32[_f32]
+  //   _za64[_f64] (only if __ARM_FEATURE_SME_F64F64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmla_lane_za32[_f32]_vg1x4(uint32_t slice, svfloat32x4_t zn,
+                                   svfloat32_t zm, uint64_t imm_idx);
+  
+  
+  // Variants are available for:
+  //   _za32[_f32]
+  //   _za64[_f64] (only if __ARM_FEATURE_SME_F64F64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmls_lane_za32[_f32]_vg1x2(uint32_t slice, svfloat32x2_t zn,
+                                   svfloat32_t zm, uint64_t imm_idx);
+  
+  
+  // Variants are available for:
+  //   _za32[_f32]
+  //   _za64[_f64] (only if __ARM_FEATURE_SME_F64F64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmls_lane_za32[_f32]_vg1x4(uint32_t slice, svfloat32x4_t zn,
+                                   svfloat32_t zm, uint64_t imm_idx);
+  
+  ```
+
+#### FMLAL, BFMLAL, SMLAL, UMLAL (single)
+
+Multi-vector multiply-add long (widening)
+
+``` c
+  // Variants are also available for _za32[_f16], _za32[_s16] and _za32[_u16]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmla[_single]_za32[_bf16]_vg2x1(uint32_t slice, svbfloat16_t zn,
```

Comment:
I'm not sure the `vg2x1` and `vg4x1` functions should have a `[_single]` suffix.  I'd understood `[_single]` to be for functions that take a tuple argument followed by a single-vector argument, to distinguish them from two tuple arguments.  That is, `[_single]` seems like it's for `x2`, `x4`, `vgNx2` and `vgNx4`.